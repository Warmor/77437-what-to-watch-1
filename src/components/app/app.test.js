import React from 'react';
import Enzyme, {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import {shallowToJson} from 'enzyme-to-json';
Enzyme.configure({adapter: new Adapter()});

import App from "~/components/app/app";

describe(`<App/>`, () => {
  it(`App renders correctly`, () => {
    const tree = shallow(<App/>);
    expect(shallowToJson(tree)).toMatchSnapshot();
  });
});
