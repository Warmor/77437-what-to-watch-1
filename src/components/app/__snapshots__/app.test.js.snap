// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> App renders correctly 1`] = `
<Fragment>
  <Switch>
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(PageWelcome)",
          "type": [Function],
        }
      }
      exact={true}
      path="/"
    />
    <ConnectFunction
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(PageMyList)",
          "type": [Function],
        }
      }
      path="/my-list"
    />
    <ConnectFunction
      component={[Function]}
      exact={true}
      path="/login"
    />
  </Switch>
</Fragment>
`;
